pkg/apis/rio.cattle.io/v1/image.go

```diff
@@ -45,23 +49,21 @@ type ImageBuildSpec struct {
 	PushRegistrySecretName string `json:"pushRegistrySecretName,omitempty"`
 
 	// Specify image name instead of the one generated from service name, format: $registry/$imageName:$revision
-	BuildImageName string `json:"buildImageName,omitempty"`
+	ImageName string `json:"imageName,omitempty"`
 
 	// Whether to enable builds for pull requests
 	PR bool `json:"pr,omitempty"`
 
-	// Whether to enable builds for pull requests
+	// Whether to enable builds for tags
 	Tag bool `json:"tag,omitempty"`
 
 	// Build image with no cache
 	NoCache bool `json:"noCache,omitempty"`
 
-	// BuildTimeout describes how long the build can run
-	BuildTimeout *metav1.Duration
-
-	// ServiceName The service to update when the build succeeds or fails.  If blank do not update any service.
-	ServiceName string `json:"serviceName,omitempty"`
+	// Timeout describes how long the build can run
+	Timeout *metav1.Duration
 }
 
 type ImageBuildStatus struct {
+	ImageName string
```

Comment:
Should this contain a field that determines if the image is done building?

Reply:
@daxmc99 I don't think darren care too much about fields in status. So add those fields if you need. Those are mostly internal types that are consumed by controllers.